var/spack/repos/builtin/packages/datatransferkit/package.py

```diff
@@ -11,14 +11,16 @@ class Datatransferkit(CMakePackage):
     parallel solution transfer services for multiphysics simulations"""
 
     homepage = "https://datatransferkit.readthedoc.io"
-    url = "https://github.com/ORNL-CEES/DataTransferKit/archive/3.1-rc3.tar.gz"
+    url = "https://github.com/ORNL-CEES/DataTransferKit/archive/3.1.1.tar.gz"
     git = "https://github.com/ORNL-CEES/DataTransferKit.git"
 
     tags = ["e4s"]
 
     maintainers("Rombur")
 
     version("master", branch="master", submodules=True)
+    version("3.1.1", commit="bfb7673cc233c26a6a541cbf096f37f26df1e5fb", submodules=True)
+    version("3.1.0", commit="60a4cbd0a55505e0450f1ac979e1eef8966dc03f", submodules=True)
     version("3.1-rc3", commit="691d5a1540f7cd42141a3b3d2a7c8370cbc3560a", submodules=True)
     version("3.1-rc2", commit="1abc1a43b33dffc7a16d7497b4185d09d865e36a", submodules=True)
```

Comment:
@Rombur do we want to deprecate these release candidates now that there are stable versions available?

Reply:
I don't think that's necessary. If I understood the doc correctly, deprecating packages should be used for insecure packages which is not the case here.

Reply:
Insecure versions are the most common reason for deprecating a package. However we [commonly deprecate/remove release candidates](https://github.com/spack/spack/pull/40141#pullrequestreview-1641076225) since they're versions that are unlikely to be used/wanted by end users.